libraries/systeminfo/src/sys_win32.cpp

```diff
@@ -1,6 +1,6 @@
 #include "sys.h"
 
-#include <windows.h>
+#include <Windows.h>
```

Comment:
I think this is a trademark violation, as `Windows` is a registered trademark of the Microsoft Corporation.

Reply:
Feature

Reply:
nooo i feel like we are doing a hostile takeover or smth

Reply:
I have gotten permission from my father (bill gates) to use this code.